src/EventSubscriber/MaintenanceEventsubscriber.php

```diff
@@ -47,13 +45,9 @@ public function handle(RequestEvent $event): void
             return;
         }
 
-        $responseContent = $this->translator->trans('maintenance.ui.message');
-
-        if ('' !== $configuration->getCustomMessage()) {
-            $responseContent = $this->twig->render('@SynoliaSyliusMaintenancePlugin/maintenance.html.twig', [
-                'custom_message' => $configuration->getCustomMessage(),
-            ]);
-        }
+        $responseContent = $this->twig->render('@SynoliaSyliusMaintenancePlugin/maintenance.html.twig', [
```

Comment:
this wasn't done in another PR ? Or maybe you merged it in this branch ?

Reply:
Yes, the PR #32 was merged on this PR.